src/linear_algebra/matrix.lean

```diff
@@ -363,20 +363,43 @@ end is_basis
 section mul_linear_map_to_matrix
 
 variables {N : Type*} [add_comm_group N] [module R N]
-variables {b : ι → M} {b' : ι' → M} {c : ι → N} {c' : ι' → N}
+variables {κ κ' : Type*} [fintype κ] [fintype κ']
```

Comment:
I'd be tempted to put this up on line 280 to keep the index variable adjacent, but it doesn't really matter